### PR TITLE
[New] Update from Bizcharts to ChartJs

### PR DESCRIPTION
**Pull request by mo-5**
#### Screenshots (if applicable)

image
image

#### Checklist
If the database has been modified:
- [ ] Update database diagram 
- [ ] Create new migration 

If API endpoints has been modified:
- [ ] Update backend documentation 
- [ ] Create/Update validators for the API endpoints 
- [ ] The API endpoints are properly secured with keycloak 

If the UI has been modified:
- [ ] It is translated in both language (with no hard coded text) 
- [ ] Has been tested on IE
- [ ] The modifications are tab friendly
- [ ] It is accessible

### CODE REVIEW

**Comment by mo-5** [comment]
before:
image

after:
image

**Comment by mo-5** [comment]
I noticed this 
image

Does that mean ant- design can be imported using es ?

**Comment by BenJeau** [comment]
@MohamedRadwan maybe, we could use the babel plugin thing, but its not working with more than just one lib right now, I tried it and don't remember if it went down that much. I think if we see `es` in the analyze means that we are using the `es`?

**Comment by mo-5** [comment]
> @MohamedRadwan maybe, we could use the babel plugin thing, but its not working with more than just one lib right now, I tried it and don't remember if it went down that much. I think if we see `es` in the analyze means that we are using the `es`?

Sounds good, I think if we see the whole package including es that means we are not using it however if we only see es that means we are using es and not importing the whole package. Thats what I understand from this

File: services/frontend/src/components/admin/dashboardGraphs/DashboardGraphsView.jsx [hunk 1]
```diff
@@ -138,7 +123,9 @@ const DashboardGraphsView = ({
               }
               loading={monthlyGrowth.length === 0}
             >
-              <Chart
+              <Chart type="line" data={monthlyGrowthData} options={options} />
+
+              {/* <Chart
```
Comment (BenJeau) [inline services/frontend/src/components/admin/dashboardGraphs/DashboardGraphsView.jsx]: Looks good, I would remove this comment and design wise for the charts, we can work on it after this is merged, unless you want to do it now

**Review by BenJeau — APPROVED** [review]

File: services/frontend/src/components/admin/dashboardGraphs/DashboardGraphsView.jsx (hunk 1 above)
Comment (mo-5) [reply services/frontend/src/components/admin/dashboardGraphs/DashboardGraphsView.jsx]: I think we can work on the design after this merge just to test out chart js first

**Pull request merged**